server/controllers/entities/lib/resolver/update_resolved_entry.js

```diff
@@ -51,3 +61,21 @@ const addImageClaim = async (entity, imageUrl, newClaims) => {
   const { url: imageHash } = await getImageByUrl(imageUrl)
   newClaims['invp:P2'] = [ imageHash ]
 }
+
+const updateDatePrecision = (entity, updatedEntity, seedClaims) => {
+  const seedDateClaims = _.pick(seedClaims, simpleDayProperties)
+
+  _.forEach(seedDateClaims, (seedDates, prop) => {
```

Comment:
[minor point: syntax]
same as above, even if here the 2 arguments bring some more value, the standard way isn't thaaaat much of a stretch:
```suggestion
  Object.keys(seedDateClaims).forEach(prop => {
    const seedDates = seedDateClaims[prop]
```
or
```suggestion
  for (const [ prop, seedDateClaims ] of Object.entries(seedDateClaims) {
```
(that last one takes a bit of getting use to, but it's standard, and learning it means you can go use that knowledge everywhere JS is accepted with zero dependency, that is, to our great joy, everywhere ;))

Reply:
`Object.keys(seedDateClaims).forEach` feels more friendly to people coming from other languages though.